Pages/Index.cshtml.cs

```diff
@@ -17,30 +17,24 @@ public class IndexModel : PageModel
     {
         public string GetData(string endpoint)
         {
-            string downloadedData = "";
```

Comment:
Refactored code

Reply:
Either way is fine here.  The return statement is usually the last statement in a method, so I prefer the old way myself.  It feels un-natural to break out of a using statement with a return statement.